src/scripts/components/Play.js

```diff
@@ -676,7 +698,46 @@ class Play extends Component<Props, State> {
               ) : null}
             </PlayerWrapper>
           </VideoWrapper>
-          {!isEmbed && <DescriptionWrapper />}
+          {!isEmbed &&
+            video && (
+              <PlayInfo>
+                {video.title && <Title small>{video.title}</Title>}
+                {video.author && <Text>By {video.author}</Text>}
+                <PlayInfoButtons>
+                  <ButtonIcon>
+                    <SVG>
+                      <use xlinkHref="#icon-play-view" />
+                    </SVG>
+                    <Text small gray>
+                      0
+                    </Text>
+                  </ButtonIcon>
+                  <ButtonIcon>
+                    <SVG>
+                      <use xlinkHref="#icon-play-like" />
+                    </SVG>
+                    <Text small gray>
+                      0
```

Comment:
Do we want to hard code in these values?

Reply:
I was thinking to create a component for our SVGs. But for now, let's keep like this.
What you think, @bent0b0x?

Reply:
I think this is fine for now. In my comment above though I was talking about the `0`s that are hard coded